bin/installer.sh

```diff
@@ -236,18 +236,18 @@ function check_snmptrapd_run() {
 		echo "Returned : $ret"
 		echo "Waiting 5 sec to come up"
 		sleep 5
-		port=$(netstat -apn |grep -E "^udp.*:162" 2>&1)
+		port=$(ss -pluna | grep ':162 ' 2>&1)
 		if [ $? -ne 0 ]; then
 			echo 'snmptrapd started but not listening to udp/162.... Exiting...'
 			return 0;
 		fi
 	fi
 	
 	# get pid 
-	snmpPid=$(echo $port | cut -d ' ' -f 6 | cut -d '/' -f1);
-	snmpName=$(echo $port | cut -d ' ' -f 6 | cut -d '/' -f2);
+	snmpPid=$( echo $port | grep -oP '(?<=pid=)[0-9]+' );
```

Comment:
man page says "This is highly experimental and grep -P may warn of unimplemented features".
Original code was ugly, so maybe : 
```suggestion
	snmpPid=$( echo $port | sed -r 's/.*pid=([0-9]+).*/\1/' );
```